legacy/app/auth/authentication.service.js

```diff
@@ -61,13 +70,68 @@ function (
     }
 
     function setToLogoutState() {
+         let refreshToken = Session.getSessionDataEntry('refreshToken');
+
+         let payload = {
+             refresh_token: refreshToken,
+             client_id: CONST.OAUTH_CLIENT_ID,
+             client_secret: CONST.OAUTH_CLIENT_SECRET
+         };
+         // Do we need to cancel the scheduled refresh?
```

Comment:
> // Do we need to cancel the scheduled refresh?
Yes I think so..  or at the very least make sure it will do nothing if it triggers